apps/block_scout_web/lib/block_scout_web/views/api/v2/fee_payment_view.ex

```diff
@@ -0,0 +1,20 @@
+defmodule BlockScoutWeb.API.V2.FeePaymentView do
+  use BlockScoutWeb, :view
+  alias Explorer.Chain.Address
+
+  def render("fee_payments.json", %{fee_payments: fee_payments, next_page_params: next_page_params}) do
+    %{"items" => Enum.map(fee_payments, &prepare_fee_payment(&1, nil)), "next_page_params" => next_page_params}
+  end
+
+  def prepare_fee_payment(fee_payments, _conn) do
```

Comment:
Nitpick: make the parameter name singular too `fee_payment`

Reply:
done